### PR TITLE
celestia: update recipe

### PR DESCRIPTION
**Pull request by TheLostProbe**
updates the homepage, copyright, and summary to be more up-to-date

also, before this gets merged, is the homepage equal to the project's main website, or its source code repository? i do not know much about haiku, im just here to make sure this unofficial port is still up-to-date. celestia is a cross-platform project, and so i believe everyone on every platform should have an equal experience with it. though in saying that, our amiga boys arent doing too good, being stuck on 1.6.0 and all that :(

### CODE REVIEW

**Comment by waddlesplash** [comment]
Hey, thanks for your contribution, and welcome!

The homepage should indeed be the main website, it's put into package metadata and displayed e.g. in the main section of HaikuDepot.

You also need to increment the `REVISION` field, as the package won't be re-built otherwise.

File: sci-astronomy/celestia/celestia-1.6.2.2.recipe [hunk 1]
```diff
@@ -11,8 +11,8 @@ simple to navigate through the universe to the object you want to visit.
 Celestia is expandable. Celestia comes with a large catalog of stars, galaxies, \
 planets, moons, asteroids, comets, and spacecraft. If that's not enough, you \
 can download dozens of easy to install add-ons with more objects."
-HOMEPAGE="http://celestia.sourceforge.net"
-COPYRIGHT="2001-2020 Celestia Development Team"
+HOMEPAGE="https://celestia.space"
+COPYRIGHT="2001-2021 Celestia Development Team"
 LICENSE="GNU GPL v2"
 REVISION="1"
```
Comment (waddlesplash) [inline sci-astronomy/celestia/celestia-1.6.2.2.recipe:17]: Change this to `2`.

**Review by waddlesplash — CHANGES_REQUESTED** [review]

File: sci-astronomy/celestia/celestia-1.6.2.2.recipe (hunk 1 above)
Comment (TheLostProbe) [reply sci-astronomy/celestia/celestia-1.6.2.2.recipe:17]: 1.6.2.2 was released in 2021 so it shouldnt be 2022(?)

**Comment by TheLostProbe** [comment]
the `REVISION` should be good now, i also re-added those gaps in the `SUMMARY` because i think it would create an issue when being displayed

**Review by waddlesplash — APPROVED** [review]

**Comment by waddlesplash** [comment]
Thanks again!

**Comment by TheLostProbe** [comment]
> BTW, in your quest to update metadata or other information everywhere... do you know about Repology? It tracks packages across distributions (mostly Linux, but HaikuPorts is in there, too.) [Here's Celestia's page](https://repology.org/project/celestia/information). You can then click over to the other tabs to see the individual package descriptions, versions, etc. across repositories.

looks neat, it sure will come in handy

> Thanks again!

no problem!

**Pull request merged**